### PR TITLE
Add one seconde to timestamp

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**

### CODE REVIEW

File: PaloAltoCortexXDR/manifest.json [hunk 1]
```diff
@@ -27,5 +27,5 @@
     "name": "Palo Alto Cortex XDR",
     "uuid": "2bb1aaf9-a90c-411d-8e5f-c72b1b46b3d7",
     "slug": "paloalto_cortex_xdr",
-    "version": "1.0.0"
+    "version": "1.2.0"
```
Comment (squioc) [inline PaloAltoCortexXDR/manifest.json:30]:
```suggestion
    "version": "1.0.1"
```

**Review by squioc — CHANGES_REQUESTED** [review]
Could you also update the changelog? Thanks.

File: PaloAltoCortexXDR/CHANGELOG.md [hunk 2]
```diff
@@ -7,8 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-- Add for the first time Cortex Query alerts
\ No newline at end of file
+- Add for the first time Cortex Query alerts
+
+## 2024-02-01 - 1.0.1
+
+### Change
+
+- Improve the timestamp setter ( Add one seconde to the last timestamp )
```
Comment (squioc) [inline PaloAltoCortexXDR/CHANGELOG.md:20]: Could you reverse the order? The most recent entry should be the first one.

**Review by squioc — CHANGES_REQUESTED** [review]

File: PaloAltoCortexXDR/CHANGELOG.md (hunk 2 above)
Comment (TOUFIKIzakarya) [reply PaloAltoCortexXDR/CHANGELOG.md:20]: yes sorry for that

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**